src/java/org/apache/cassandra/db/compaction/OperationType.java

```diff
@@ -26,6 +26,7 @@
     CLEANUP("Cleanup", true, 1),
     SCRUB("Scrub", true, 1),
     UPGRADE_SSTABLES("Upgrade sstables", true, 1),
+    DOWNGRADE_SSTABLES("Downgrade sstables", true, 1),
```

Comment:
perhaps not important in this case, but we should always add enums at the end because otherwise, if they are serialized as numbers somewhere, that can break the serialization